contrib/mathtex-script-type/Makefile

```diff
@@ -1,9 +0,0 @@
-.PHONY: build integrity
-
-build: $(BUILDDIR)/contrib/mathtex-script-type.min.js
-
-$(BUILDDIR)/contrib/mathtex-script-type.min.js: $(BUILDDIR)/mathtex-script-type.js
-	$(UGLIFYJS) < $< > $@
-
-$(BUILDDIR)/mathtex-script-type.js: mathtex-script-type.js
```

Comment:
Is outputting `mathtex-script-type.js` to `$(BUILDDIR)`, not `$(BUILDDIR)/contrib`, and not including in the distribution archive an intended behavior?

Reply:
Good catch.